signature-aggregator/aggregator/aggregator.go

```diff
@@ -121,6 +121,14 @@ func (s *SignatureAggregator) CreateSignedMessage(
 		s.metrics.FailuresToGetValidatorSet.Inc()
 		return nil, fmt.Errorf("%s: %w", msg, err)
 	}
+	s.metrics.ConnectedStakeWeightPercentage.WithLabelValues(
+		signingSubnet.String(),
+	).Set(
+		float64(
+			connectedValidators.ConnectedWeight /
+				connectedValidators.TotalValidatorWeight,
```

Comment:
```suggestion
		float64(connectedValidators.ConnectedWeight) /
		float64(connectedValidators.TotalValidatorWeight),
```

This way we actually get float division, instead of doing integer division and converting to float